pkg/apis/core/types.go

```diff
@@ -1627,6 +1627,9 @@ type CSIVolumeSource struct {
 	// +optional
 	ReadOnly *bool
 
+	// DEPRECATED. Kubernetes cannot translate FSType into something that the driver's
```

Comment:
The issue you referenced has been closed as being factually incorrect.

Deprecating parts of the api is a bigger deal than just adding a comment like this, it generally requires discussion and buy-in from the SIGs that own the API (in this case, storage and/or node).

Reply:
Turns out that I didn't subscribe to #79986 - hence didn't get the notice from the github

Reply:
No worries, no one can watch all the stuff that happens here. :)

I wouldn't take the lesson "watch everything" from this, rather "think carefully about whether people are giving good advice" :)